src/components/CodeEditor.svelte

```diff
@@ -63,13 +63,13 @@
     min-height: 350px;
     max-height: 700px;
     width: 100%;
-    display: flex;
+    display: flex; /* stylelint-disable-line plugin/no-unsupported-browser-features */
```

Comment:
Enable flexbox with options instead of suppressing the error on every line

```javascript
"plugin/no-unsupported-browser-features": [true, {
  ignore: ["flexbox"]
}]
``` 

https://github.com/ismay/stylelint-no-unsupported-browser-features#options